pages/linux/lvremove.md

```diff
@@ -0,0 +1,11 @@
+# lvremove
+
+> Remove one or more logical volumes.
+
+- Remove a logical volume in a volume group:
+
+`sudo lvremove {{volume_group}}/{{logical_volume}}`
+
+- Remove all logical volumes in a volume group:
+
+`sudo lvremove {{volume_group}}`
```

Comment:
```suggestion
`lvremove {{volume_group}}`
```

Reply:
I think it should stay here, @navarroaxel. Generally we include a `sudo` prefix if a command requires it to function.